### ISSUE TITLE
List dependencies

### ISSUE THREAD

**Issue by Myoldmopar** [Myoldmopar/GroundHeatFlowsPaper#2]
Should be pretty straightforward, dependencies are gnuplot, and texlive-something.  We could add specific tex live packages to the dependencies as needed.

**Comment by mitchute** [comment]
Ressurecting this project?

**Comment by Myoldmopar** [comment]
Well, always on the lookout for papers.  And I have some thoughts about a ground heat transfer paper.

**Comment by mitchute** [comment]
Yeah, I'd love to get some credit for working on this (in the form of a paper). Let me know if/when you want to get started, though, I will say that I won't be much help until October-ish.
